### PR TITLE
Styles token update

### PR DESCRIPTION
**Pull request by MaryGodservant**
## Checklist before requesting a review

- [x] I have applied `percy` label for o-[COMPONENT] or `chromatic` label for o3-[COMPONENT] on my PR before merging and after review. Find more details in [CONTRIBUTING.md](https://github.com/Financial-Times/origami/blob/main/CONTRIBUTING.md#pull-requests-and-visual-regression-tests)
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.
- [ ] I have updated relevant env variables in Doppler.

### CODE REVIEW

**Comment by frshwtr** [comment]
Note: this will need to be a breaking change as the token names have changed.

**Comment by frshwtr** [comment]
Looks good, let me know once you've committed the changes to typography based on the decisions we made earlier and i'll give the final review.

File: components/o3-foundation/src/css/components/Typography/body.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+.o3-typography-wrapper > p {
+	font-family: var(--o3-typography-use-case-body-standard-font-family);
+	font-weight: var(--o3-typography-use-case-body-standard-font-weight);
+	font-size: var(--o3-typography-use-case-body-standard-font-size);
+	line-height: var(--o3-typography-use-case-body-standard-line-height);
+	margin: 0 0 var(--o3-spacing-s);
+	color: var(--o3-color-use-case-body-text);
+}
+
+.o3-typography-body--small {
```
Comment (notlee) [inline components/o3-foundation/src/css/components/Typography/body.css:11]: This isn't a modifier `o3-typography-body o3-typography-body--small`, instead it should be `o3-typography-body-small`

Comment (notlee) [inline components/o3-foundation/src/css/components/Typography/body.css:11]: I think there's a large/big style too?

Comment (notlee) [inline components/o3-foundation/src/css/components/Typography/body.css:11]: FYI I noticed the prop on the story only makes some of the body copy small

**Comment by notlee** [comment]
Added a commit to fix some inverse styles (incorrect custom prop names) and ensured the inverse theme applies to new body classes – not confident I caught each case, any way we can increase confidence with visual regression tests in the future?

**Review by notlee — APPROVED** [review]

**Pull request merged**